src/Makefile.am

```diff
@@ -80,7 +80,7 @@ EXTRA_DIST = \
 
 CLEANFILES = *~ *\# .\#* *.py?
 
-FLAKE8_IGNORE = E126,E127,E128,E201,E202,E203,E211,E221,E222,E225,E226,E231,E241,E251,E261,E262,E265,E266,E301,E302,E303,E305,E306,E402,E501,E502,W503,W504,E722,E741
+FLAKE8_IGNORE = E126,E127,E128,E201,E202,E203,E211,E221,E222,E225,E226,E231,E241,E251,E261,E262,E265,E266,E301,E302,E303,E305,E306,E402,E501,E502,E722,E741,W503,W504
```

Comment:
Can we remove this since you added a `.flake8` ? i.e. `flake8 --config $(top_src_dir)/.flake8`

Reply:
yes, I don't think it requires `--config`, it should fine the file automatically.

Also, then the `.flake8` file should be dist-ed, so that the test works with the tarball too...

changed.